docs/videos/healthcare-and-life-sciences/hcls-blueprint.yaml

```diff
@@ -0,0 +1,356 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+blueprint_name: hcls-cluster
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: hcls-01
+  region: us-central1
+  zone: us-central1-c
+
+  network_name: hcls-cluster-net
+  subnetwork_name: primary-subnet
+  bucket_name_software: user-provided-software
+  bucket_name_inputs: input-bucket
+  bucket_name_outputs: output-bucket
+  use_deployment_name_in_bucket_name: false
+  homefs_server_ip:  # Set on command line
+  appsfs_server_ip:  # Set on command line
+
+deployment_groups:
+- group: enable_apis
+  modules:
+
+  ### Enable APIs ###
+
+  - id: services-api
+    source: community/modules/project/service-enablement
+    settings:
+      gcp_service_list:
+      - file.googleapis.com
+      - iam.googleapis.com
+      - pubsub.googleapis.com
+      - secretmanager.googleapis.com
+      - serviceusage.googleapis.com
+      - compute.googleapis.com
+      - stackdriver.googleapis.com
+
+- group: setup
+  modules:
+
+  ### Network ###
+
+  - id: network-create
+    source: modules/network/vpc
+
+  ### Resource Monitoring ###
+
+  - id: hpc-dash
+    source: modules/monitoring/dashboard
+
+  ### Storage ###
+
+  - id: homefs-create
+    source: modules/file-system/filestore
+    use: [network-create]
+    settings: {name: home}
+
+  - id: appsfs-create
+    source: modules/file-system/filestore
+    use: [network-create]
+    settings: {name: apps}
+
+  - id: bucket-software-create
+    source: ./community/modules/file-system/cloud-storage-bucket
+    settings:
+      name_prefix: $(vars.bucket_name_software)
+
+  - id: bucket-input-create
+    source: ./community/modules/file-system/cloud-storage-bucket
+    settings:
+      name_prefix: $(vars.bucket_name_inputs)
+
+  - id: bucket-output-create
+    source: ./community/modules/file-system/cloud-storage-bucket
+    settings:
+      name_prefix: $(vars.bucket_name_outputs)
+
+- group: software_installation
+  modules:
+
+  - id: network-install
+    source: modules/network/pre-existing-vpc
```

Comment:
This may look like magic to people. I would add a comment saying that this would be reuse the network created in the previous group, or something like that.

Reply:
done.